framework/PostProcessors/BasicStatistics.py

```diff
@@ -345,6 +363,22 @@ def __computeVp(self,p,weights):
     vp = vp.sum()
     return vp
 
+  def __computeEquivalentSampleSize(self,weights):
+    """
+      Compute the equivalent sample size for given probability weights
+      @ In, weights, xarray.Dataset, probability weights of all input variables
+      @ Out, equivalentSize, xarray.Dataset, the equivalent sample size
```

Comment:
the equivalentSize is a xarray.Dataset, the keys are the variable names. This is because, all the probability weights are stored as Dataset, and all the calculations are now based on Dataset. and the calculation results are also Dataset now.